.github/workflows/development.yml

```diff
@@ -0,0 +1,28 @@
+name: Development Workflow
+
+on:
+  push:
+    branches: [ '**' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+
+  build-and-test:
+
+    runs-on: ubuntu-latest
+    environment: SNOWSHU_DEVELOPMENT
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Setup credentials
+      env:
+        CREDENTIALS: ${{ secrets.SNOWFLAKE_CREDENTIALS }}
+      run: echo "$CREDENTIALS" | base64 --decode > tests/assets/integration/credentials.yml
```

Comment:
For now, we should probably skip the integration tests in github CI, and worry about setting envs and secrets later on.